docker-compose.circle2.yml

```diff
@@ -140,3 +160,18 @@ centos7test:
     - st2-packages-vol
   volumes:
     - /sys/fs/cgroup:/sys/fs/cgroup
+
+centos8test:
+  image: stackstorm/packagingtest:centos8-systemd
+  privileged: true
+  cap_add:
+    - SYS_ADMIN
+  security_opt:
+    - seccomp:unconfined
+  extends:
+    file: docker-compose.override.yml
```

Comment:
Coompare with https://github.com/StackStorm/st2-packages/pull/590/files#diff-3af181705858a30af4e3d8752a16d414R126 diff.

It looks like `centos8test` shouldn't extend `file: docker-compose.override.yml`.